src/pages/Login/Login.js

```diff
@@ -1,5 +1,91 @@
-import React from 'react';
+/* eslint-disable */
 
-export default function Login() {
-  return <div>hi</div>;
+import React, { useRef, useState } from 'react';
+import { useNavigate } from 'react-router';
+import '../Login/Login.scss';
+import Input from './Input/Input';
+
+function Login() {
+  const idInputRef = useRef();
+  const pwInputRef = useRef();
+  const loginData = {};
+  const [isAlertPopId, setIsAlertPopId] = useState(false);
+  const [isAlertPopPw, setIsAlertPopPw] = useState(false);
+
+  const validateIdBtn = () => {
+    loginData['id'] = idInputRef.current.value;
+    const idValue = loginData['id'];
+    const isIdValid =
+      idValue.includes('@') && idValue.length > 2 ? true : false;
```

Comment:
```suggestion
      idValue.includes('@') && idValue.length > 2
```
이미 비교식의 결과값이 true or false이므로 굳이 삼항연산자로 다시 true or false값을 만들어줄 필요가 없습니다!